# RCC-spatial-mapping
### Mapping single cell transcriptomes in the intra-tumoural and associated territories of kidney cancer

Code relating to https://www.cell.com/cancer-cell/fulltext/S1535-6108(22)00548-7. Detailed method and use of this code and dataset in the above article.

Tumour behaviour is intricately dependent on the oncogenic properties of cancer cells and their multi-cellular interactions. To better understand these dependencies within the wider micro-environment, we studied over 270,000 single cell transcriptomes and 100 micro-dissected whole exomes from 12 patients with kidney tumours, prior to validation through the use of spatial transcriptomics. Tissues were sampled from multiple regions of the tumour core, the tumour-normal interface, normal surrounding tissues, and peripheral blood. We found the tissue-type location of CD8+ T cell clonotypes largely defined their exhaustion state, with intra-tumoural spatial heterogeneity that is not well-explained by somatic heterogeneity. De novo mutation calling from single cell RNA sequencing data allows us to broadly infer the clonality of stromal cells and lineage-trace myeloid cell development. We discovered six conserved meta-programmes that distinguish tumour cell function. An epithelial-mesenchymal transition meta-programme, highly enriched at the tumour-normal interface appears co-localised with IL1B expressing macrophages, which could represent a plausible therapeutic target.
